src/components/dynamic/user/Header.jsx

```diff
@@ -26,15 +26,17 @@ const Header = ({ horizontal = true }) => {
         horizontal ? "flex-row" : "flex-col"
       } items-center justify-center w-fit`}
     >
-      <Image
-        src={session.user.image}
-        width={125}
-        height={125}
-        alt="Picture of user's profile"
-        className="mr-4 rounded-full overflow-hidden"
-      />
-      <div className="align-left">
-        <p className="text-2xl font-bold mb-0">{session.user.name}</p>
+      <div className="border-[1px] border-white rounded-full p-1 w-fit mr-4">
```

Comment:
this can be combined cause u can apply padding on the image, make the image background white, and then add margin + border